wandb/sdk/data_types/object_3d.py

```diff
@@ -161,6 +218,47 @@ def __init__(
         else:
             raise ValueError("data must be a numpy array, dict or a file object")
 
+    @classmethod
+    def from_file(cls, data_or_path: Union[io.StringIO, "FileFormat3D"]) -> "Object3D":
+
+        return cls(data_or_path)
+
+    @classmethod
+    def from_numpy(cls, data: "np.ndarray") -> "Object3D":
+        if not util.is_numpy_array(data):
+            raise ValueError("`data` must be a numpy array")
+
+        if len(data.shape) != 2 or data.shape[1] not in {3, 4, 6}:
+            raise ValueError(
+                """The shape of the numpy array must be one of either
+                                [[x y z],       ...] nx3
+                                 [x y z c],     ...] nx4 where c is a category with supported range [1, 14]
+                                 [x y z r g b], ...] nx4 where is rgb is color"""
+            )
+
+        return cls(data)
+
+    @classmethod
+    def from_point_cloud(
+        cls,
+        points: Sequence["Point"],
+        boxes: Sequence["Box3D"],
+        vectors: Optional[Sequence["Vector3D"]] = None,
+        point_cloud_type: "PointCloudType" = "lidar/beta",
+        # camera: Optional[Camera] = None,
+    ) -> "Object3D":
+        if point_cloud_type not in get_args(PointCloudType):
```

Comment:
@dmitryduev  this check fails in unit tests where I pass in a valid point_cloud_type.  Error is: `ValueError: Point cloud type not supported`  

Note that PointCloudType is not in quotes like the other instances of PointCloudType - when I try to add quotes, since it's getting treated as a regular string. the get_args docs I found weren't super helpful.

Reply:
This was happening bc it was only defined when you run type checking. I replaced with a more verbose check due to additional problems with py36.